tt_metal/jit_build/data_format.cpp

```diff
@@ -173,6 +176,9 @@ void check_valid_in_out_data_formats(DataFormat input_formats[NUM_OPERANDS], Dat
         TT_FATAL(is_exp_b_format(last_valid_input_format) == is_exp_b_format(last_valid_intermed_format),
             "Input format = {} and Intermed format = {} must have same exp width", last_valid_input_format, last_valid_intermed_format);
     }
+
+    DataFormat last_out_valid_format = check_valid_formats_within_operand(output_formats);
+    TT_FATAL(last_out_valid_format != DataFormat::Invalid, "Output format not found");
```

Comment:
Are the TT_FATALs checking user supplied values or internal values?  If user supplied, this is good, if internal consistency then they should be TT_ASSERTs

Reply:
The `unpack_src_format` and `pack_dst_format` are technically user supplied. `unpack_dst_format` & `pack_src_format` are internally supplied. I can change the fatal to asserts for the latter formats